collectors/carbonblack/collector.js

```diff
@@ -0,0 +1,161 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * carbonblack class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+const calcNextCollectionInterval = require('@alertlogic/paws-collector').calcNextCollectionInterval;
+const parse = require('@alertlogic/al-collector-js').Parse;
+const utils = require("./utils");
+
+
+let typeIdPaths = [];
+
+let tsPaths = [];
+
+
+class CarbonblackCollector extends PawsCollector {
+
```

Comment:
could you please add a simple constructor to pass package.json version into it.
For example, see [here](https://github.com/alertlogic/paws-collector/blob/master/collectors/auth0/auth0_collector.js#L32-L34)

Reply:
sure